docs/section5/Rule5-10.md

```diff
@@ -0,0 +1,31 @@
+
+# Envelope - Rule 5-10  
+
+**Rule ID:** 5-10  
+**Rule Description:**  Fenestration for new buildings, existing buildings, and additions shall be assumed to be flush with the exterior wall, and no shading projections shall be modeled.  
+**Rule Assertion:** Baseline RMR = expected value  
+**Appendix G Section:** Section G3.1-5(d) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
+
+- For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
+
+  - For each zone in thermal block: ```for zone_baseline in thermal_block_baseline.zones:```  
+
+  - For each space in thermal zone: ```for space_baseline in zone_baseline.spaces:```  
+
+    - Get surface in space: ```for surface_baseline in space_baseline.surfaces:```  
+
+      - Get vertical glazings in exterior wall: ```if ( surface_baseline.classification == "WALL" ) AND ( surface_baseline.adjacent_to == "AMBIENT" ): windows_baseline = surface_baseline.fenestration_subsurfaces```  
+
+      **Rule Assertion:** Baseline vertical fenestrations are flush with the exterior wall, and no shading projects shall be modeled: ```(window.has_shading_overhang == FALSE) AND (window.has_shading_sidefins == False) for window in windows_baseline```  (Note XC, any data elements for flush with the exterior wall?)
```

Comment:
I think having no overhang and no side fins is as close to "flush with exterior wall" that we are going to get.

Reply:
@JasonGlazer yes, I agree. This RDS looks good to me.

Reply:
@xingcx and @JasonGlazer similar to the discussion on the RDS related to manual interior shades, we should also issue a warning if shading projections are modeled in the baseline

Reply:
@supriyagoel , I think this should be an error, not a warning. I don't think there are any circumstances when baseline has projections. 
@xingcx , I think surface should be classified as a wall based on its tilt. I wouldn't use surface_baseline.classification for anything unless we first verify that surfaces are correctly classified as walls or roofs based on 90.1 definition.

Reply:
I agree with @supriyagoel that this should be an error and not a warning. Otherwise looks good to me

Reply:
Updated. Changed to raise_error.